### DISCUSSION TITLE
Integrating ripgrep or ag into the terminal?

### DISCUSSION THREAD

**Discussion by lispercat** [antonmedv/llama#106] [Ideas]
For me it would be wonderful to have some file search with text integrated.
Than it would make the terminal experience complete.

**Comment by antonmedv** [comment]
But you already can use ripgrep?

**Reply by lispercat** [reply]
yes, I can! So there is already a way to call it from walk?

**Reply by lispercat** [reply]
I guess I am missing a way to get to the command line while I am in walk
